inflect.py

```diff
@@ -1398,7 +1398,8 @@ def _get_value_from_ast(self, obj):
             return tuple([self._get_value_from_ast(e) for e in obj.elts])
 
         # None, True and False are NameConstants in Py3.4 and above.
-        elif sys.version_info.major >= 3 and isinstance(obj, ast.NameConstant):
+        elif sys.version_info.major >= 3 and \
+                isinstance(obj, ast.NameConstant):  # pragma: no cover
```

Comment:
Why exclude Python 3 from coverage?

We are running tests on Python 2 and 3, both with coverage.

And we can see from the previous commit that these lines are covered by some of the builds.

https://coveralls.io/builds/19276359/source?filename=inflect.py#L1401

The coverage is combined at the end, so we want as much coverage as possible, from all tested versions.

Reply:
My bad. Never used travis or coveralls. I looked at the last build and assumed it only runs 2.7 based on the highlighted line. I'll revert the change.

Coverage check seems to fail because of these two conditions. Should I just ignore these?

Reply:
Yeah, it's not always clear what causes coverage change! Especially for bigger changes like this.

Master:
1168 of 1179 relevant lines covered (99.07%)

PR:
29 of 30 new or added lines in 1 file covered. (96.67%)
1144 of 1155 relevant lines covered (99.05%)

I think that "29 of 30" might be a red herring, and coverage dropped because the total number of lines dropped.

If we look at the coverage for the previous commit: https://coveralls.io/builds/19276359/source?filename=inflect.py

The first non-covered line is 1682:
https://coveralls.io/builds/19276359/source?filename=inflect.py#L1682

But this PR doesn't change anything after line 1475, so I don't think we need to worry about it for this PR. Just revert a9fef1efa334739544ace1c6fe49956fa8145e41.

(Having said that, the 12 or so uncovered lines could probably be tested fairly easily, but perhaps it's better to keep that separate.)